tools/html-version/bounded-context-canvas-template.html

```diff
@@ -351,5 +351,73 @@ <h1>Bounded Context Canvas (v5)</h1>
                 </section>
             </article>
         </footer>
+
+        <script type="module">
```

Comment:
We don't _have_ to use ESM (i.e. require modern browsers), but I figured that's a reasonable constraint in this context.